cves/CVE-2019-3498.yml

```diff
@@ -49,7 +49,9 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  By specifically crafting a url, an attacker could cause spoofed information 
+  appear on the Django default 404 webpage
```

Comment:
Can you expand this? How would the spoofed information be supplied through the URL (i.e as a param or something else?). Confirm if the spoofed information persists for every user and if an XSS is possible (I'm guessing it's not).

Reply:
Django themselves actually didn't expand on it. But I'll see if I can add something.